examples/chunkhash/README.md

```diff
@@ -95,12 +95,12 @@ module.exports = {
 /******/ 		var result;
 /******/ 		for(var i = 0; i < deferredModules.length; i++) {
 /******/ 			var deferredModule = deferredModules[i];
-/******/ 			var fullfilled = true;
+/******/ 			var fulfilled = true;
```

Comment:
Fix the source code instead: https://github.com/webpack/webpack/blob/master/lib/web/JsonpMainTemplatePlugin.js#L258